packages/core/src/components/DatePicker/DatePicker.tsx

```diff
@@ -171,6 +181,8 @@ const DatePicker = forwardRef(
       setFocusedInput(focusedInput || FocusInput.startDate);
     }, []);
 
+    const mergedPhrases = { ...DayPickerRangeController.defaultProps.phrases, ...phrases };
```

Comment:
This will break in React 19 as `defaultProps` is removed, is there a different way?

Reply:
I've added default values as constants, since react-dates doesn't expose them at all.

Reply:
you're right, they expose it, but it's not typed! nice find – I've updated it.